### DISCUSSION TITLE
Regularlab Modules Anywhere does not work with Astroid Template

### DISCUSSION THREAD

**Discussion by Lflorijn** [templaza/astroid-framework#344] [General]
I want to use Regularlabs module anywhere. But it doesn't work with the Astroid Template. Is there an alternative or do I have to set something.

**Comment by Chacapamac** [comment]
I @Lflorijn 
I just test **Regularlabs Module Anywhere** on my Astroid Template One / Joomla 4 test website and it seem to work correctly?
http://joom4-astro.graf-test.ca/

![Screen Shot 2023-06-02 at 10 09 18 AM](https://github.com/templaza/astroid-framework/assets/1907276/5571ad11-166b-4537-a49b-c00874ad10af)

•Be sure that your module is “**Enabled**” and you got a proper “**Menu Assignment**” on your Module
